### DISCUSSION TITLE
toyhammeronabyssmonster event can only be trigger by hitting vanilla abyss monsters

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15022] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

```xml
<StatusEffect type="OnUse" target="UseTarget" delay="1" comparison="Or">
  <Conditional speciesname="latcher"/>
  <Conditional speciesname="endworm"/>
  <Conditional speciesname="charybdis"/>
  <TriggerEvent identifier="toyhammeronabyssmonster" />
</StatusEffect>
```
if other mods add their own abyss monster you are less likely meet vanilla ones
maybe use Conditional group="abyss" ?

### Reproduction steps

1. set campaigndata pathofthebikehorn as 2
2. use toy hammer to hit modded abyss creatures
3. you cant trigger the event

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/be48d7963b1e4e064790f48a2de9bb28c6b9ccd9 (commit in our private repo). Fix coming up in a future update.
